docker/Dockerfile.sys-mgmt-agent

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2018
+# Dell
+# Cavium
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+FROM golang:1.11.2-alpine3.7 AS builder
+WORKDIR /go/src/github.com/edgexfoundry/edgex-go
+
+# The main mirrors are giving us timeout issues on builds periodically.
+# So we can try these.
+RUN echo http://nl.alpinelinux.org/alpine/v3.7/main > /etc/apk/repositories; \
+    echo http://nl.alpinelinux.org/alpine/v3.7/community >> /etc/apk/repositories
+
+RUN apk update && apk add make glide git
+COPY . .
+
+RUN make prepare
+RUN make cmd/sys-mgmt-agent/sys-mgmt-agent
+
+#Next image - Copy built Go binary into new workspace
+FROM alpine:3.7
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+      copyright='Copyright (c) 2018: Dell, Cavium'
+
+# Layer-in the Docker-in-Docker image now...
+FROM docker:dind
+
+COPY --from=builder /go/src/github.com/edgexfoundry/edgex-go/cmd/sys-mgmt-agent/sys-mgmt-agent /
+COPY --from=builder /go/src/github.com/edgexfoundry/edgex-go/cmd/sys-mgmt-agent/res/configuration.toml /res/configuration-docker.toml
+
+RUN apk --no-cache add curl
+# Do we (still) need the following directive (for adding 'make')?
+# RUN apk --no-cache add make
```

Comment:
Uncomment and change this to `RUN apk --no-cache add py-pip`.  You will not need make anymore here I think.

Reply:
Excellent point! **Done**